pre-commit

```diff
@@ -82,40 +82,41 @@ check() {
     fi
 }
 
-check "Benchmark compilation" \
-    cargo build --benches --no-default-features --features "enable-benches compact-gate"
-
 check "Clippy checks" \
-    cargo clippy --tests -- -D warnings
+    cargo clippy --features="cli test-fixture" --tests -- -D warnings
 
-check "Clippy concurrency checks" \
-    cargo clippy --tests --features shuttle -- -D warnings
+check "Tests" \
+    cargo test --features="cli test-fixture"
 
-check "Clippy web checks" \
-    cargo clippy --tests --no-default-features --features "cli web-app real-world-infra test-fixture compact-gate" -- -D warnings
+if [ -z "$EXEC_SLOW_TESTS" ]
```

Comment:
this file really contains only one change - lifts `-z $SLOW_TESTS` up higher to hide all extra checks by default